cell_feature_data/README.md

```diff
@@ -0,0 +1,46 @@
+# Cell Feature Data
+
+[![Dataset validation](https://github.com/allen-cell-animated/cell-feature-data/actions/workflows/validate.yml/badge.svg?branch=main)](https://github.com/allen-cell-animated/cell-feature-data/actions/workflows/validate.yml)
```

Comment:
more badges(e.g. license, build, version, etc) will be added once merged to `main`